src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -152,11 +179,36 @@ public final void refreshItems() {
 
     }
 
+    private int getClosedIssuesCount() {
+        return (int) getElementsList().stream().filter((element) -> !element.getIssue().isOpen()).count();
+    }
```

Comment:
There are 2 levels of data.

The lower level data would be in `MultiModel` i.e. `Logic#models`, suitable for stats across repositories or in a `Model` in `models` if the stats concern only 1 repository. This might not be a suitable place for the method as a panel displays mixed subsets of issues across multiple repositories depending on that panel's filter.

The higher level data is a `List` of `GuiElements` that represents the data-state of a panel and is stored within the `ListPanel` object itself. Ideally, we can have a class that encapsulate this, which would become a good place to put this method.